examples/http-certification/custom-assets/README.md

```diff
@@ -383,22 +388,56 @@ fn certify_index_asset() {
 }
 ```
 
+It's also possible to skip certification for certain routes. This can be relatively easily as follows:
+
+```rust
+const DYNAMIC_REQ_PATH: &str = "/dynamic";
+
+fn certify_dynamic_asset() {
```

Comment:
It's really super confusing, because actually we add a certification for this path that tells the gateway to skip verification for the response. So there is actually some verification steps involved to make sure that it is not skipped arbitrarily.

I used `add_certification_skips` instead, which I think is clearer, let me know if it's still confusing.